ammico/summary.py

```diff
@@ -169,99 +236,364 @@ def load_vqa_model(self):
         )
         return summary_vqa_model, summary_vqa_vis_processors, summary_vqa_txt_processors
 
-    def analyse_image(self):
+    def analyse_image(
+        self,
+        analysis_type: Optional[str] = None,
```

Comment:
why do we pass all of these here? Are they not given at class instantiation?

I guess this is because of the "asking question, and then another question" option? I would only pass the absolutely necessary keywords and leave everything else at the instantiation (`__init__`).